scripts/qa_crowbarsetup.sh

```diff
@@ -5055,16 +5186,17 @@ function oncontroller_testpreupgrade
     openstack stack create upgrade_test -t \
         /root/scripts/heat/2-instances-cinder.yaml \
         --parameter image=$tempest_image $heat_stack_params
-    wait_for 15 20 "openstack stack list | grep upgrade_test | grep CREATE_COMPLETE" \
-             "heat stack for upgrade tests to complete"
+    wait_for 15 20  "openstack stack list | grep upgrade_test | grep CREATE_COMPLETE" \
+                    "heat stack for upgrade tests to complete"
```

Comment:
Why is the whitespace changed here?

Reply:
Bashate complained about the indent on line 5090 and I indented rather than outdented (and adjusted line 5189 for optical alignment). If you like I can move it the other way...

Reply:
I'm surprised there should be any complain, this should be unchanged part of the code (save the whitespace)